### ISSUE TITLE
Fails to compile on ampere altra

### ISSUE THREAD

**Issue by 0xAlcibiades** [DaniPopes/keccak-asm#4]
```
keccak1600-armv8.s:209: Error: invalid operands (*ABS* and *UND* sections) for '*'
```
etc etc

```
Error: undefined symbol __SIZEOF_POINTER__ used as an immediate value
```

etc etc

in `keccak1600-armv8.s`

Processor is arm v8.2

**Comment by DaniPopes** [comment]
if this is linux arm, we don't support this yet. I have not figured out a way to fix this, see for example #2, #3.
In #2 it compiles but fails tests

**Comment by 0xAlcibiades** [comment]
> if this is linux arm, we don't support this yet. I have not figured out a way to fix this, see for example #2, #3. In #2 it compiles but fails tests

Is this resolved now?

**Comment by DaniPopes** [comment]
Yes
